### PR TITLE
feat: api landing page favico, misc fixes

### PR DESCRIPTION
**Pull request by kilrau**
Closes https://github.com/BoltzExchange/api-landing/issues/2

### CODE REVIEW

File: lib/api/static/api.html [hunk 1]
```diff
@@ -41,10 +42,10 @@
...
-  <h1>Privacy First, Non-Custodial Bitcoin Exchange</h1>
-  <p>Can't be evil. You are in control of your money. Always.</p>
+  <h1>API Endpoint</h1>
```
Comment (michael1011) [inline lib/api/static/api.html:45]: I don't like this heading. Looks a little out of place

File: lib/api/static/api.html [hunk 2]
```diff
@@ -41,10 +42,10 @@
...
+  <a class="btn" target="_blank" href="https://docs.boltz.exchange/v/api/">API Docs</a>
+  <a class="btn" target="_blank" href="https://api.boltz.exchange/getpairs">API Test</a>
```
Comment (michael1011) [inline lib/api/static/api.html:47]: It's also in the docs and seems redundant here

**Review by michael1011 — CHANGES_REQUESTED** [review]

File: lib/api/static/api.html (hunk 1 above)
Comment (kilrau) [reply lib/api/static/api.html:45]: Heading as in *different heading style* or *wording*?

File: lib/api/static/api.html (hunk 2 above)
Comment (kilrau) [reply lib/api/static/api.html]: The `API Test` button? I knew you wouldn't. Removed.

**Pull request merged**